test/e2e/antreapolicy_test.go

```diff
@@ -102,6 +102,7 @@ func initialize(t *testing.T, data *TestData) {
 	p80 = 80
 	p81 = 81
 	p5000 = 5000
+	p6000 = 6000
```

Comment:
maybe these should be called `portUDP` and `portSCTP` instead

By the way, I think you can use the same port numbers for UDP, TCP and SCTP so maybe there's no need for these new port numbers (5000, 6000).

Reply:
BTW maybe we should have a test where we allow TCP traffic on port X and deny UDP traffic on port X, and check that the policy is enforced correctly when the port is the same

Reply:
Good point. Done.